core-concepts/fees.md

```diff
@@ -65,20 +65,20 @@ These fees incentivize individuals and organizations to run their provider insta
-| <b>Order Fees <br>1 datatoken available to get dataset acces   |                                           |                                                                                                                                                   |
-| publishMarket<br>Absolute value, in any token. E.g. 5 USDT     |                     0                     | Set in market config, by the publishing market.                                                                                                   |
-| consumeMarket<br>Absolute value, in any token. E.g. 2 DAI      |                     0                     | Set in market config, by the consuming market.                                                                                                    |
-| Ocean Community<br>Fixed price in DT                           |                  0.03 DT                  | 0.03 DT                                                                                                                                           |
-
-| Ocean Provider Fees                                         |      OPF Provider      | 3rd party Provider             |
+| <b>Swap Fees</b>                                           | <b>Value in Ocean Market, using any Provider</b> | <b>Value in Other Markets</b>                                                    |
+| :--------------------------------------------------------- | :----------------------------------------------: | :------------------------------------------------------------------------------- |
+| publishMarket: FixedRate                                   |                        0%                        | Set in the market config, by the publishing market.<br>Min = 0.001%<br>Max = 50% |
+| consumeMarket: FixedRate<br>ERC20Template                  |                        0%                        | 0%                                                                               |
+| consumeMarket: FixedRate<br>EnterpriseTemplate             |                        0%                        | Set in market config, by the consuming market.                                   |
+| Ocean Community: FixedRate<br>OCEAN, H2O as base token     |                       0.1%                       | 0.1%                                                                             |
+| Ocean Community: FixedRate<br>other base token             |                       0.2%                       | 0.2%                                                                             |
+| <b>Publish Fees</b>                                        |                        0%                        | 0%                                                                               |
```

Comment:
I don't really get why `Publish Fees` and `Order Fees` are in bold?

Reply:
- [ ] Split the 2 tables in 4 tables.

Reply:
@akshay-ap are you still working on this or do you want me to take it over?

Reply:
preview here: https://app.gitbook.com/o/mTcjMqA4ylf55anucjH8/s/Z7MCljB2mgupjI8thP3H/~/revisions/mwFN7QXAK52pNcfqqv8z/core-concepts/fees#fee-values